mobile/src/people/employees-list-item.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { TouchableHighlight, StyleSheet, Platform, Text, View, Image } from 'react-native';
+
+import { Avatar } from '../people/avatar';
+import { Employee } from '../reducers/organization/employee.model';
+import { employeesListItemStyles as styles } from './styles';
+import { StyledText } from '../override/styled-text';
+
+interface EmployeesListItemProps {
+    employee: Employee;
+    id: string;
+}
+
+export class EmployeesListItem extends React.Component<EmployeesListItemProps> {
+    public render() {
+        const employeeName = this.props.employee ? this.props.employee.name : 'Unknown';
```

Comment:
Not `Unknown`. null

Reply:
Are there any employee in a db without a name?

Reply:
should not be any. I think such employees fill throw an exception on deserialization

Reply:
@victor-kotov-arcadia this comment is still not addressed